src/main/java/com/programmers/application/controller/voucher/command/VoucherExecutionDependencyInjector.java

```diff
@@ -0,0 +1,36 @@
+package com.programmers.application.controller.voucher.command;
+
+import com.programmers.application.controller.voucher.command.execution.CreateVoucherExecution;
+import com.programmers.application.controller.voucher.command.execution.ExitVoucherExecution;
+import com.programmers.application.controller.voucher.command.execution.ListVoucherExecution;
+import jakarta.annotation.PostConstruct;
+import org.springframework.stereotype.Component;
+
+@Component
+public class VoucherExecutionDependencyInjector {
+
+    private final CreateVoucherExecution createVoucherExecution;
+    private final ExitVoucherExecution exitVoucherExecution;
+    private final ListVoucherExecution listVoucherExecution;
+
+    public VoucherExecutionDependencyInjector(CreateVoucherExecution createVoucherExecution, ExitVoucherExecution exitVoucherExecution, ListVoucherExecution listVoucherExecution) {
+        this.createVoucherExecution = createVoucherExecution;
+        this.exitVoucherExecution = exitVoucherExecution;
+        this.listVoucherExecution = listVoucherExecution;
+    }
+
+    @PostConstruct
+    public void inject() {
+        for (Command command : Command.values()) {
+            if (command.equals(Command.CREATE)) {
```

Comment:
```suggestion
            if (command == Command.CREATE) {
```

Enum인 싱글턴이라 일반적으로 `==`을 사용해 동일성 비교를 합니다.
동등성 비교와 동일성 비교에 대해서 알아보는 것도 좋겠네요